README.md

```diff
@@ -47,158 +62,69 @@ just docker-build astria-sequencer debug
 
 ## Running locally
 
-The entire stack consists of 6 different binaries. It's recommended to use the setup located in [astriaorg/dev-cluster](https://github.com/astriaorg/dev-cluster), but running everything manually is documented here as well.
-
-To run the entire stack locally, you will additionally need cometbft installed, which requires that [Go](https://go.dev/doc/install) is installed.
-
-The binaries required are as follows:
-- `astria-sequencer`
-- `cometbft`
-- `astria-sequencer-relayer`
-- Celestia
-- `astria-conductor`
-- Astria's `go-ethereum` fork
-
-The first two binaries act as the sequencer chain, while go-ethereum (plus conductor) acts as the "rollup"; it executes sequenced transactions that are destined for that rollup.
-
-#### Install cometbft
-Ensure `~/go` is in your `PATH`, or `GOPATH` is set to some other place in your `PATH`.
-
-```sh
-git clone https://github.com/astriaorg/cometbft
-cd cometbft
-export GOPATH=~/go
-make install
-```
-
-#### Start the sequencer chain
-
-First, start `astria-sequencer`:
-
-```sh
-./target/debug/astria-sequencer --genesis-file=crates/astria-sequencer/test-genesis.json
-```
-
-Then, start cometbft:
-```sh
-cometbft init
-# set block time to 15s 
-sed -i'.bak' 's/timeout_commit = "1s"/timeout_commit = "15s"/g' ~/.cometbft/config/config.toml
-cometbft start
-```
+The entire stack consists of many different binaries. It's recommended to use the
```

Comment:
Note this is actual content change. The directions were no longer accurate and didn't pass formatting rules in many places.

Reply:
should we move it somewhere after it's updated? or just remove lol

Reply:
I can work on updating it if that's helpful, I think it would be

Reply:
I don't think we want to actually have this documented long term. We should instead work on making dev cluster the easiest way to test code.

Running microservice clusters all on bare metal same machine is a long way from the production environment, and more likely to test locally say "it works" when it won't in production.

Admittedly we have some work to do on dev-cluster to make this the reality, so I'm torn on documenting it right now.